qcodes/dataset/measurements.py

```diff
@@ -1220,8 +1220,6 @@ def set_shapes(self, shapes: Shapes) -> None:
             shapes: Dictionary from names of dependent parameters to a tuple
                 of integers describing the shape of the measurement.
         """
-        RunDescriber._verify_interdeps_shape(interdeps=self._interdeps,
```

Comment:
were is this done now? if at all?

Reply:
This is not done anymore since this only checked that `not (num_setpoints > dims) ` Since we now allow scanning multiple parameters in parallel that check does not make sense anymore